package.json

```diff
@@ -49,6 +50,8 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
+    "hygen": "^6.1.0",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
What is this package needed for?

Reply:
importing scss 🤔 may not need this now that not importing via js; will retry

Reply:
updated, good call